roseau/load_flow/models/transformers/transformers.py

```diff
@@ -73,47 +73,58 @@ def __init__(
 
         self.tap = tap
         # Compute the phases if not provided, check them if provided
-        w1_has_neutral = "n" in parameters.winding1.lower()
-        w2_has_neutral = "n" in parameters.winding2.lower()
+        w1_has_neutral = "y" in parameters.winding1.lower() or "z" in parameters.winding1.lower()
+        w2_has_neutral = "y" in parameters.winding2.lower() or "z" in parameters.winding2.lower()
```

Comment:
The check was made on the presence of `"n"` because `TransformerType` defines `y` or `z` alone as "without neutral" and `yn` & `zn` as "with a neutral". Doesn't the new check break this assumption?

Reply:
The good variable name could be `w1_accepts_neutral_point`. If you have a Wye or Zig-zag transformer, there is one.

With this change, we accept that a winding "y" and "yn" are the same. (idem for "z" and "zn"). The connection of the neutral point is fully determined by the `phase1` argument of the constructor (and `phase2` for the second winding). It might lead to a simplification of the `TransformerParameters` class in order to accept `y` and `yn` but keeping internally `"y"` as `winding` field in both case.